Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -830,6 +836,8 @@ public class DynamicHlsController : BaseJellyfinApiController
             [FromRoute, Required] string playlistId,
             [FromRoute, Required] int segmentId,
             [FromRoute, Required] string container,
+            [FromQuery, Required] long runtimeTicks,
+            [FromQuery, Required] long actualSegmentLengthTicks,
```

Comment:
This would be a breaking change to the API

Reply:
Indeed it would, but we're also responsible for creating the URL.